### DISCUSSION TITLE
Help for Responsive Web Design

### DISCUSSION THREAD

**Discussion by nev-apongshine22** [community/community#83070] [Programming Help]
### Body

Hey guys i 'll like you help me please make my website that is irresponsive to become responsive without neccesarily changing the content but doing simple modifications to it with their respective comments . The code is below
 
HTML:
`<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <link rel="stylesheet" href="LaWeb-Home.css">
    <link rel="stylesheet" href="css/all.min.css">
    <link rel="stylesheet" href="css/all.css">
    <link rel="stylesheet" href="css/brands.css">
    <link rel="stylesheet" href="css/brands.min.css">
    <link rel="stylesheet" href="css/fontawesome.css">
    <link rel="stylesheet" href="css/fontawesome.min.css">
    <link rel="stylesheet" href="css/regular.css">
    <link rel="stylesheet" href="css/regular.min.css">
    <link rel="stylesheet" href="css/solid.css">
    <link rel="stylesheet" href="css/solid.min.css">
    <link rel="stylesheet" href="css/svg-with-js.css">
    <link rel="stylesheet" href="css/svg-with-js.min.css">
    <link rel="stylesheet" href="css/v4-font-face.css">
    <link rel="stylesheet" href="css/v4-font-face.min.css">
    <link rel="stylesheet" href="css/v4-shims.css">
    <link rel="stylesheet" href="css/v4-shims.min.css">
    <link rel="stylesheet" href="css/v5-font-face.css">
    <link rel="stylesheet" href="css/v5-font-face.min.css">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>LaWeb | Home</title>
</head>
<body>

<header class="header_1">
<a href="#" class="reg_">Registration</a>
<a href="#" class="contact_">Contact</a>
<button class="get_start" href="#"><i class="fa-solid fa-house-chimney" style="color: #ffffff;"></i> In / Up</button>
</header>

<div class="bg_img">
<span class="linear_text_gradient">The institution</span>
<span class="linear_tex_gradient">for you</span><button class="start"><i class="fa-solid fa-house" style="color: #ffffff;"></i> Get Started</button>
</div>

<center><h1>Our Patners</h1></center> 
<marquee>
  <img src="las4s7yy.png" height="150px" width="150px" style="margin-right:200px;">
   <img src="km8zz8p7.png" height="150px" width="150px" style="margin-right:200px;">
    <img src="5di6g1j2(1).png" height="150px" width="150px" style="margin-right:200px;">
 </marquee>

<div class="what">
  <center> <h1>What we offer</h1></center>
   <center> <p class="offer"> Lorem ipsum, dolor sit amet consectetur adipisicing Lorem ipsum dolor sit amet
     consectetur adipisicing elit. Eum eligendi animi dolorem reiciendis
     qui inventore libero! Cupiditate,Lorem, ipsum dolor sit amet consectetur adipisicing elit.
      Voluptates repellat natus iusto soluta consectetur ipsum minus
      a rem at, exercitationem tenetur, corporis quos asperiores. 
    Repudiandae eius esse, alias obcaecati
 </p></center>
   
     <div class="container">
        <div class="container_1">
        <center> <h4>Learning in absolute conditions of your choice </h4></center>
        <p><center>Schulz Web provides learners with any constructional learning program
           of their choice managing their time </center></p>
        </div>
        <div class="container_2">
        <center><h4>Free formation of members on particular carreers</h4></center>
        <p><center>Lorem ipsum dolor sit amet consectetur adipisicing elit. Fuga soluta sapiente libero quaerat
            blanditiis repellendus architecto? </center></p>
        </div>
        <div class="container_3">
        <center><h4>Develop your learning capacities up to standard in an eye blink</h4></center>
        <p><center>Fuga soluta sapiente libero quo earum
        placeat modi autem,consecteur dis ascipeng alias eligendi? 
        voluptas in dicta cumque neque,</center></p>    </div>
        </div> 
        
    <center> <h2>General Courses</h2></center>
  <div class="gen_course">
    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Sunt asperiores nihil explicabo delectus officia,
    laborum adipisci beatae minima rerum dolores sit nemo,
    iure nostrum possimus velit voluptatem in quaerat impedit!</p>
    <div class="course_type">
    <div class="course1">
    <center> <p>Computer Science</p> </center> 
      <span  class="c5">Lorem ipsum dolore sit amet, consectetur adipisicing elit. Eum sit odit veritatis tenetur, id libero, 
       </span>
    </div>
    <div class="course2">
    <center>  <p>Language and Literature</p></center> 
      <span  class="c5">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Eum sit odit veritatis tenetur, id libero, 
       !</span>
    </div>
    <div class="course3">
    <center>  <p>Mathematics</p></center> 
      <span  class="c5">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Eum sit odit veritatis tenetur, id libero, 
       </span>
    </div>
    <div class="course4">
    <center>  <p>Food Science</p></center> 
      <span  class="c5">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Eum sit odit veritatis tenetur, id libero, 
       </span>
    </div>
    <div class="course5">
    <center>  <p>Accounting $ Finance</p> </center> 
      <span class="c5">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Eum sit odit veritatis tenetur, id libero, 
       </span>
    </div>
    <div class="course6">
    <center> <p>Geography</p></center> 
      <span  class="c5">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Eum sit odit veritatis tenetur, id libero, 
       </span>
    </div>
    
  </div>
    
  <center><ul class="linear_text_gradient">Offered Subjects</ul></center>
    <li> <i class="fa-solid fa-check" style="color: #1100ff;"></i> Mathematics</li>
    <li> <i class="fa-solid fa-check" style="color: #1100ff;"></i> English Language</li>
    <li> <i class="fa-solid fa-check" style="color: #1100ff;"></i> French Language</li>
    <li> <i class="fa-solid fa-check" style="color: #1100ff;"></i> Biology</li>
    <li> <i class="fa-solid fa-check" style="color: #1100ff;"></i> Chemistry</li>
    <li> <i class="fa-solid fa-check" style="color: #1100ff;"></i> Computer Science</li>

   </div>

  <center> <h2>Our Certifications</h2>
<div class="certifying">
    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit.</p>
    <p> Sunt asperiores nihil explicabo delectus officia,
    laborum adipisci beatae minima rerum dolores sit nemo,
    iure nostrum possimus velit voluptatem in quaerat impedit!</p>
</div><center>

</div>

<center><h1>Our Administrators and Teachers</h1></center> 

<div class="o_teachers">
 <div class="ad_1">
    <img alt="" src="" class="adm_1">
    <center>  <span>Mr NDA DENNIS</span>
  <h5>PROPIETOR</h5></center> 
</div>

<div class="ad_2">
    <img alt="" src="" class="adm_2">
    <center>  <span>Mme NDA GRACE</span>
  <h5>PROPIETRESS</h5></center> 
</div>

<div class="ad_3">
    <img alt="" src="" class="adm_3">
    <center> <span>Mr NJOUNY JULUIS</span>
  <h5>PRINCIPAL</h5></center> 
</div>

<div class="ad_4">
    <img alt="" src="" class="adm_4">
    <center>  <span>Mr TEH AFUKEZIR</span>
  <h5>DEAN OF STUDIES</h5></center> 
</div>
 
<div class="ad_5">
    <img alt="" src="" class="adm_5">
    <center>  <span>Mr NTUNGWE NESTOR</span>
  <h5>SENIOR DISCIPLINE MASTER</h5> </center> 
</div>

<div class="ad_6">
    <img alt="" src="" class="adm_6">
    <center>  <span>Mr BEKINDAKA DAVID</span>
  <h5>COMPUTER SC. / ICT TEACHER</h5></center> 
</div>

</div>

</body>
</html>`

CSS:
`*{
    padding: 0em;
    margin: 0em;
}
body{
    max-height:100%;
    max-width:100%;
    scale:100%;
    min-width:100%;
    max-width:100%;
    background-repeat:no-repeat;
    overflow-x:hidden;
    font-family: system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
}
.header_1{
    height: 50px;
    width: 100%;
    display: flex;
    justify-content: space-between;
    flex-direction: row;
    flex-wrap: nowrap;
    justify-content: flex-end;
    align-items:center;
    background: whitesmoke;
}
 .reg_{
    font-family:system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
    font-size:14px;
    font-weight:500;
    color:blue;
    text-decoration:none;
}
 .contact_{
    font-family:system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
    font-size:14px;
    font-weight:500;
    margin-left: 5%;
    color:blue;
    text-decoration:none;
}
 .get_start{
    border:none;
    height:70%;
    width: 10%;
    border-radius: 15px;
    margin-left: 5%;
    margin-right: 5%;
    background: blue;
    font-size:14px;
    font-weight:bolder;
    color:aliceblue;
    text-decoration:none;
}
.get_start:hover{
    scale: 105%;
    transition: 2s;
}
.bg_img{
    height: 90vh;
    background-repeat: no-repeat;
    background-size: cover;
    font-size: 30px;
    display:flex;
    flex-direction: column;
    font-weight: 900;
    background: url("LaWeb3.jpg") ;
    color:aliceblue;
    width: 100%;
}
.linear_text_gradient{
    background: linear-gradient(to right, blue, red);
    -webkit-background-clip: text;
    background-clip: text;
    color: transparent;
  }
  .linear_tex_gradient{
    background: linear-gradient(to right, rgb(153, 0, 255), red);
    -webkit-background-clip: text;
    background-clip: text;
    color: transparent;
  }
.bg_img span{
    font-size:100px;
    margin-top:4%;
    margin-left:10%;
    background-repeat: no-repeat;
    font-family: system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
    font-weight: 900;
}
.start{
    border:none;
    height:50px;
    width: 15%;
    border-radius: 15px;
    margin-left: 5%;
    margin-right: 5%;
    margin-bottom:10%;
    background: blue;
    font-size:14px;
    font-weight:bolder;
    color:aliceblue;
    text-decoration:none;
}
.start:hover{
    scale: 105%;
    transition: 2s;
}
.container{
    display:flex;
    flex-direction: row;
    flex-wrap:wrap;
    justify-content: initial;
    vertical-align: auto;
    font-weight: 500;
    margin-top: 3%;
    margin-left: 30% 30% 30% ;
}
.container_1{
    height: 10em;
    width: 27.5%;
    display: flex;
    justify-content: initial;
    flex-wrap: wrap;
    vertical-align:auto;
    flex-direction: row;
    border: 3px solid rgb(17, 0, 255);
    margin-left:4.5%;
    border-radius: 6px;
}
.container_2{
    height: 10em;
    width: 27.5%;
    display: flex;
    justify-content: initial;
    flex-wrap: wrap;
    vertical-align:auto;
    flex-direction: row;
    border: 3px solid rgb(21, 228, 243);
    margin-left: 5%;
    border-radius: 6px;
}
.container_3{
    height: 10em;
    width: 27.5%;
    display: flex;
    justify-content: initial;
    flex-wrap: wrap;
    vertical-align:auto;
    flex-direction: row;
    border: 3px solid  rgb(6, 87, 107);
    margin-left: 5%;
    border-radius: 6px;
}
.container_1:hover{
    transform: scale(105%);
    transition: 2s;
}
.container_2:hover{
    transform: scale(105%);
    transition: 2s;
}
.container_3:hover{
    transform: scale(105%);
    transition: 2s;
}
.course_type{
    display: flex;
    flex-direction: row;
    flex-wrap: wrap;
    background:no-repeat;
    align-items: center;
}

.course1{
    height: 15em;
    width: 25%;
    margin-top: 3%;
    margin-left: 4%;
    margin-right: 4%;
    border: none;
    border-radius:5%;
    background: url("Computer-Science.jfif");
}
.course2{
    height: 15em;
    width: 25%;
    margin-top: 3%;
    margin-left: 4%;
    margin-right: 4%;
    border: none;
    border-radius:5%;
    background: url("L-L.jfif");
}
.course3{
    height: 15em;
    width: 25%;
    margin-left: 3%;
    margin-top: 4%;
    margin-right: 4%;
    border: none;
    border-radius:5%;
    background: url("Science.jfif");
}
.course4{
    height: 15em;
    width: 25%;
    margin-left: 4%;
    margin-top: 3%;
    margin-right: 4%;
    border: none;
    border-radius:5%;
    background: url("Food science.jpg");
}
.course5{
    height: 15em;
    width: 25%;
    margin-left: 4%;
    margin-top: 3%;
    margin-right: 4%;
    border: none;
    border-radius:5%;
    background: url("Acc-Econs.jfif");
}
.course6{
    height: 15em;
    width: 25%;
    margin-left: 4%;
    margin-top: 3%;
    margin-right: 4%;
    border: none;
    border-radius:5%;
    background: url("Phys-Educ.jfif");
}
.course_type p{
    font-family: system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
    font-size: 4em;
    justify-content: center;
    color:whitesmoke;
    margin-left: 3%;
    font-weight: 500;
}
.course_type span{
    height:3%;
    max-width: 14%;
    color:white;
    font-weight: 500;
    font-family: system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
}

ul{
    list-style: none;
    font-size: 35px;
    font-weight: bolder;
    margin-bottom: 3%;
}
li{
    list-style: none;
    margin-left: 10%;
    font-size: 20px;
    font-weight:600;
    margin-right: 5%;
    font-family:system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
    font: 500 14px; 
}

p{
    font-family:system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
    font: 500 14px;
}
h4{
    color: cornflowerblue;
    font-weight: bolder;
}
span{
    font-family:system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
    font: 500 14px;  
}
h1{
    color:rgb(2, 40, 255);
    display:flex;
    font-family:system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
    font-weight: bolder;
    font-size:50px;
    justify-content: center;
    align-items:center;
    margin-top:50px;
    font-family: system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
}
h2{
    color:rgb(0, 166, 243);
    display:flex;
    font-size:40px;
    font-weight:bolder;
    justify-content: center;
    align-items:center;
    font-family: system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
}
h5{
    color:rgb(0, 255, 255);
    justify-content: center;
    font-weight: bold;
    font-family: system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
}
.offer{
    font-size: 14px;
    margin-right:1%;
    margin-left:1%;
    font-weight: 500;
    font-family: system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
}
.gen_course p{
   margin-right:1%;
   margin-left:1%;
   font-size: 14px; 
   font-weight:500;
}
.certifying p{
    margin-right:1%;
    margin-left:1%;
    font-size: 14px; 
    font-weight:500;
 }

.o_teachers{
    display: flex;
    flex-flow:row wrap;
    flex-direction: row;
    flex-wrap: wrap;
    align-items: center;

}
 .ad_1{
   height: 500px;
   width: 500px; 
   border: none;
   margin:10%;
   border-radius: 10%;
   align-items: center;
   background: whitesmoke;
 }
 .ad_2{
    height: 500px;
    width: 500px; 
    border: none;
    margin:10%;
    border-radius: 10%;
    align-items: center;
    background: whitesmoke;
  }
 .ad_3{
    height: 500px;
    width: 500px; 
    border: none;
    margin:10%;
    border-radius: 10%;
    align-items: center;
    background: whitesmoke;
  }
  .ad_4{
    height: 500px;
    width: 500px; 
    border-radius: 10%;
    border: none;
    margin:10%;
    align-items: center;
    background: whitesmoke;
  }
  .ad_5{
    height: 500px;
    width: 500px; 
    border: none;
    margin:10%;
    border-radius: 10%;
    align-items: center;
    background: whitesmoke;
  }
  .ad_6{
    height: 500px;
    width: 500px; 
    border: none;
    margin:10%;
    border-radius: 10%;
    align-items: center;
    background: whitesmoke;
  }
.o_teachers span{
    font-weight: 400;
    color: blue;
    float:bottom;
    font-family:system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
    font-size: 17px;

}
.adm_1{
    background-image: url("LaWeb3.jpg");
    border-radius: 50%;
}
.adm_2{
    background-image: url("LaWeb3.jpg");
    border-radius: 50%;
}
.adm_3{
    background-image: url("LaWeb3.jpg");
    border-radius: 50%;
}
.adm_4{
    background-image: url("LaWeb3.jpg");
    border-radius: 50%;
}
.adm_5{
    background-image: url("LaWeb3.jpg");
    border-radius: 50%;
}
.adm_6{
    background-image: url("LaWeb3.jpg");
    border-radius: 50%;
}`

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by davea42** [comment]
Probably meant Partners, not Patners (sorry about the insignificant nature of this comment).

**Reply by nev-apongshine22** [reply]
Thanks but that's not what i needed men

**Comment by ghost** [comment]
You could use @ media queries to make your website responsive,  also add html, css document seperately like this

```html
!DOCTYPE html
<html>
.......
```

```css
body {
}
......
```

**Reply by ghost** [reply]
This would ease out the work for others to contribute/help.
